### PR TITLE
get market info

### PR DESCRIPTION
**Pull request by StanChe**

### CODE REVIEW

File: services/cmc/main.go [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+type tezosMarketData struct{
+	Tezos MarketData `json:"tezos"`
+}
+// CoinGecko is market data provider.
+type CoinGecko struct{}
+
+// GetTezosMarketData gets the tezos price and price change from CoinGecko API.
+// TODO: some caching layer should be implemented.
+func (CoinGecko) GetTezosMarketData() (md models.MarketInfo, err error) {
+	url := "https://api.coingecko.com/api/v3/simple/price?ids=tezos&vs_currencies=usd&include_market_cap=false&include_24hr_vol=false&include_24hr_change=true&include_last_updated_at=true"
```
Comment (bolgovr) [inline services/cmc/main.go]: I think it would be better to have this in config

File: services/cmc/market_data.go [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package cmc
+
+// MarketData is a Price and Price Change with json deserialization.
+type MarketData struct{
+	Price  float64  `json:"usd"`
+	Price24hChange float64`json:"usd_24h_change"`
+}
+
+// GetPrice returns the price in USD.
+func (md *MarketData) GetPrice() float64{
```
Comment (bolgovr) [inline services/cmc/market_data.go]: nitpick but I think it would be better to reflect somehow in method name that it returning usd price

**Review by bolgovr — CHANGES_REQUESTED** [review]

File: services/cmc/main.go (hunk 1 above)
Comment (StanChe) [reply services/cmc/main.go]: Not sure about the config. It's so specific at the moment that changing the endpoint without changing the code seams not feasible. I'm thinking of 
`const baseURL = "https://api.coingecko.com/api/v3"`
and then `url := fmt.Sprintf(....` the rest of the query. What do you think?

Comment (bolgovr) [reply services/cmc/main.go]: I think just move it to const would be enough, without formatting 👍

Comment (StanChe) [reply services/cmc/main.go]: done  :)

**Review by bolgovr — APPROVED** [review]

**Pull request merged**